### PR TITLE
Feature/free 67 fe frontpage updates

### PR DESCRIPTION
**Pull request by Mitso**
Note: made FE content update 
@miltontony  @saeedmb8

### CODE REVIEW

**Comment by Mitso** [comment]
@miltontony 
Probably  something to be aware of on tickets issue;

Error
screen shot 2016-06-22 at 5 02 51 pm

**Comment by miltontony** [comment]
@Mitso that's because this repo only has a `master` branch - meaning you'll only be able to do `-b master` instead of develop 😞

File: html/index.html [hunk 1]
```diff
@@ -34,10 +34,17 @@
       <header>
         <h2>A Mobile Technology Incubator for Good</h2>
       </header>
-      <p>The Praekelt Foundation Incubator for Free Basics is a unique program designed to help developers and organisations build for the Free Basics platform of social change organisations through Internet.org and the Free Basics Platform.<br><br>Praekelt and Facebook will be onboarding 100 social change organisations using the Incubator, which will pave the way for thousands more to build for Free Basics.</p>
+      
+      <p>We believe in free access to basic life enhancing information.<br/>
+      Our technology & tools allow anyone to create and share information & services, with the potential to reach over 1 billion people.</p><br/>
```
Comment (miltontony) [inline html/index.html]: I think these (&) should be `&amp;`

**Comment by Mitso** [comment]
@miltontony thank you!

**Comment by Mitso** [comment]
Review changes:

Old:
screen shot 2016-06-23 at 9 50 43 am

Update:
screen shot 2016-06-23 at 9 50 26 am

Old:
screen shot 2016-06-23 at 9 50 49 am

Update:
screen shot 2016-06-23 at 9 50 14 am

**Comment by peterp** [comment]
@Mitso please add some padding on the paragraph to tighten it up a bit, those sentences are a bit too wide to read comfortably.

**Comment by Mitso** [comment]
@peterp please review:

screen shot 2016-06-23 at 11 31 54 am

screen shot 2016-06-23 at 11 32 32 am

screen shot 2016-06-23 at 11 33 11 am

**Comment by peterp** [comment]
👍  🍰

**Pull request merged**